src/internal/packager/helm/common.go

```diff
@@ -88,7 +88,7 @@ func loadChartFromTarball(chart v1alpha1.ZarfChart, chartPath string) (*chart.Ch
 	// Get the path the temporary helm chart tarball
 	sourceFile := StandardName(chartPath, chart) + ".tgz"
 
-	// Load the loadedChart tarball
+	// LoadState the loadedChart tarball
```

Comment:
There's several comments where Load changes to LoadState from a find and replace

Reply:
Oh man, I keep doing this 🤣 I try to scan through the diff visually before commit, but it's still easy to miss stuff. Fixed in latest